### PR TITLE
Create recipe for basic collection

### PR DESCRIPTION
**Pull request by wjthomas9**

### CODE REVIEW

File: _src/index.md [hunk 1]
```diff
@@ -12,7 +12,6 @@ There's a ton of great <a href="https://11ty.dev/">Eleventy</a> starter kits out
 
 ## Coming soon
 
-* Collections
```
Comment (peruvianidol) [inline _src/index.md:15]: Great catch here! :)

File: _src/recipes/012-create-a-collection.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+Hello from {% raw %}{{ title }}{% endraw %}!
+```
+
+4. By simply adding a **tag** to each entry, you have established your first named collection. You can now output it by using a loop, and by referencing **tag** you added to each file.
+
+```md
+## My posts
+
+{% raw %}{%- for post in collections.general %}
+  [{{ post.data.title }}]({{ post.url }})
```
Comment (peruvianidol) [inline _src/recipes/012-create-a-collection.md]: I think having these as list items would be better than paragraphs.

File: _src/recipes/012-create-a-collection.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+</p>
+```
+
+5. Once you create your first collection, there are several ways you can manipulate it...
+
+* [Excluding entries from a collection](https://www.11ty.dev/docs/collections/#option-exclude-content-from-collections)
+* [Outputting entries that share multiple tags](https://www.11ty.dev/docs/collections/#multiple-tags-single-line)
+* Custom [sorting](https://www.11ty.dev/docs/collections/#sorting) and [filtering](https://www.11ty.dev/docs/collections/#getfilteredbytag(-tagname-)) for a collection
+
+### Related
```
Comment (peruvianidol) [inline _src/recipes/012-create-a-collection.md:68]: Great idea!

File: _src/recipes/012-create-a-collection.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+</p>
+<p>
+  <a href="/posts/my-second-post.md">My Second Post</a>
+</p>
+<p>
+  <a href="/posts/my-third-post.md">My Third Post</a>
+</p>
+```
+
+5. Once you create your first collection, there are several ways you can manipulate it...
```
Comment (peruvianidol) [inline _src/recipes/012-create-a-collection.md]: Change ellipsis to a colon.

**Comment by peruvianidol** [comment]
Nice work! Thanks for submitting this!

**Comment by wjthomas9** [comment]
@peruvianidol Just made your suggested tweaks.

**Comment by peruvianidol** [comment]
Thanks again, Jon!

**Pull request merged**